packages/carbon-graphs/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 * Added
   * Added allowCalibration support for the x-axis on a line graph.
   * Finished implementation for `config.axis.x.allowCalibration` for a paired result graph with a numerical x-axis.
+  * Finished implementation for `config.axis.x.allowCalibration` for a paired result graph with a timeseries x-axis.
```

Comment:
I would combine this into a single line for simplicity.

Reply:
[4c8e03c](https://github.com/cerner/terra-graphs/pull/291/commits/4c8e03ca2d92598497fd00558d6cf42c33da90b8)